scripts/start.js

```diff
@@ -72,6 +73,20 @@ function clearConsole() {
   process.stdout.write('\x1bc');
 }
 
+function getExistingProcesses(port) {
+  //determine if another process running on the desired port
+  return new Promise(function (resolve, reject) {
+    var command = 'lsof -P | grep TCP | grep :' + port + ' | cut -d \' \' -f 1';
```

Comment:
For windows support, let's make it work on one platform and have the infra in place so that someone using windows can chime in and make it work over there with a good user experience!